core/backend/src/TextAnnotationElement.ts

```diff
@@ -0,0 +1,101 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Elements
+ */
+
+import { GeometryParams, GeometryStreamBuilder, TextAnnotation, TextAnnotation2dProps, TextAnnotation3dProps } from "@itwin/core-common";
+import { IModelDb } from "./IModelDb";
+import { AnnotationElement2d, GraphicalElement3d } from "./Element";
+import { produceTextAnnotationGeometry } from "./TextAnnotationGeometry";
+import { Id64String } from "@itwin/core-bentley";
+
+function updateAnnotation(element: TextAnnotation2d | TextAnnotation3d, annotation: TextAnnotation, subCategory: Id64String | undefined): boolean {
+  const builder = new GeometryStreamBuilder();
+
+  const params = new GeometryParams(element.category, subCategory);
+  if (!builder.appendGeometryParamsChange(params)) {
+    return false;
+  }
+
+  const props = produceTextAnnotationGeometry({ iModel: element.iModel, annotation });
+  if (!builder.appendTextBlock(props)) {
+    return false;
+  }
+
+  // ###TODO will placement bounding box be computed for me on insert/update?
```

Comment:
@bbastings questions for you.

Reply:
> @bbastings questions for you.

Yup, native code will update placement bbox.